napalm_junos/junos.py

```diff
@@ -331,6 +331,11 @@ def get_environment(self):
         for sensor_object, object_data in environment.items():
             structured_object_data = {k: v for k, v in object_data}
 
+            if structured_object_data['class']:
+                current_class = structured_object_data['class']
+            else:
+                structured_object_data['class'] = current_class
```

Comment:
Where does `current_class` come from -- doesn't seem to be defined before this test block?